search-sentiments-map/public/index.html

```diff
@@ -47,6 +52,7 @@ <h2 id="title">Sentiment Map for Google Search</h2>
       <div class="row">
         <div class="col-sm-9">
           <h5 id="topic-header">Worldwide sentiments</h5>
+          <h6 id="extrema-sentiment">Most Negative</h4>
```

Comment:
This holds both positive and negative so maybe say that? Or actually, should this have no text until it is filled with countries?